src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,11 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
-    public String getRandomColor() {
-        return null;
+
+    public ColorOfBall getRandomColor() {
+        int index = new Random().nextInt(ColorOfBall.values().length);
```

Comment:
Creating a new instance of Random every time this method is called is unnecessary and inefficient. Instead, you should create a single Random instance as a class field and reuse it.